R/print.summary.coxph.penal.R

```diff
@@ -30,12 +30,15 @@ print.summary.coxph.penal <-
 		       format(round(print1[,5], 2)),
 		       format(signif(print1[,6], 2)))
     temp <- ifelse(is.na(print1), "", temp)
-    dimnames(temp) <- dimnames(print1)
+    dimnames(temp) <- list(substring(rownames(print1), 1, maxlabel),
+			   colnames(print1))
```

Comment:
I kept `dimnames()` to minimize change, but maybe `rownames(temp) <- substring(rownames(print1), 1, maxlabel)` is enough?